workflow-templates/terraform.yml

```
@@ -1,16 +1,35 @@
 name: Terraform
-
 on:
   pull_request:
   push:
     branches:
       - $default-branch
-
+  workflow_dispatch:
+    inputs:
+      apply:
+        description: Whether to apply changes to the Terraform workspace. If false, the workflow will print any planned changes but will not apply them.
+        default: false
+        required: false
+        type: boolean
 permissions:
+  checks: write
   contents: read
-
+  id-token: write
 jobs:
   check:
-    uses: takescoop/terraform-check-modules-workflow/.github/workflows/check.yml@v1
+    uses: takescoop/terraform-check-modules-workflow/.github/workflows/check.yml@v2
     secrets:
-      terraform_token: ${{ secrets.TF_TOKEN_GET_MODULES }}
+      terraform_token: ${{ secrets.TFC_TOKEN_GET_MODULES }}
+  workspace:
+    if: false # Remove this line to enable Terraform Cloud workspaces
+    uses: TakeScoop/terraform-cloud-workspace-github-workflow/.github/workflows/terraform-workspace.yml@v1
+    secrets:
+      terraform_token: ${{ secrets.TFC_TOKEN_WORKSPACE_MANAGEMENT }}
+    with:
+      # Set working_directory to the directory containing your Terraform code.
+      working_directory: ./
```

```suggestion
      working_directory: terraform/
```

If we're going to include environments in the default workflow, I think it probably makes sense to use `terraform/` as the default working directory. Most of our users will be using environments with this working directory set